data/json/items/armor.json

```diff
@@ -9571,15 +9570,12 @@
     "encumbrance": 10,
     "material_thickness": 4,
     "use_action": {
-      "type": "holster",
-      "holster_prompt": "Stash javelins",
-      "holster_msg": "You stash your %s.",
-      "multi": 5,
-      "max_volume": "1250 ml",
-      "draw_cost": 30,
-      "flags": [ "JAVELIN" ]
+      "type": "bandolier",
+      "capacity": 5,
+      "ammo": [ "javelin", "fishspear" ],
+      "draw_cost": 30
     },
-    "flags": [ "WATER_FRIENDLY", "OVERSIZE", "BELTED" ]
+    "flags": [ "WATER_FRIENDLY", "OVERSIZE" ]
```

Comment:
Is this intended? Some kind of bag which covers torso would most likely be the same layer as a backpack (therefore the BELTED flag).